web/src/utils/helpers.ts

```diff
@@ -240,6 +242,7 @@ export const hasMobileUserAgent = () =>
 
 export const isValidHistoricalTimeRange = (timeRange: TimeRange) =>
   historicalTimeRange.includes(timeRange);
+export const memoizedIsValidHistoricalTimeRange = memoize(isValidHistoricalTimeRange);
```

Comment:
nit: It's already done so it's fine, but another time I'd probably not rename the exported function but keep that the same - to me, `isValidHistoricalTimeRange` is what the user should care about and the fact that it is memoized is an internal implementation detail that is not relevant to expose for something like this :)